arm image builds appear to be broken

It appears that the arm image builds fail like so:

```
docker build -t quay.io/kubernetes-service-catalog/service-catalog-arm:v0.1.0 ""build/"service-catalog""/tmp"

Sending build context to Docker daemon 52.57 MB

Step 1/4 : FROM arm32v7/debian:jessie

---> b0adc604307c

Step 2/4 : RUN export DEBIAN_FRONTEND=noninteractive && apt-get update && apt-get install ca-certificates -y && rm -rf /var/lib/apt/lists/*

---> Running in 1b1757bf42ad

standard_init_linux.go:178: exec user process caused "exec format error"

The command '/bin/sh -c export DEBIAN_FRONTEND=noninteractive && apt-get update && apt-get install ca-certificates -y && rm -rf /var/lib/apt/lists/*' returned a non-zero code: 1

make[1]: *** [service-catalog-image] Error 1

make[1]: Leaving directory `/home/travis/build/kubernetes-incubator/service-catalog'

make: *** [release-push-arm] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -59,19 +59,7 @@ PLATFORM?=linux
 ARCH?=amd64
 
 # TODO: Consider using busybox instead of debian
-ifeq ($(ARCH),amd64)
-	BASEIMAGE?=debian:jessie
-else ifeq ($(ARCH),arm)
-	BASEIMAGE?=arm32v7/debian:jessie
-else ifeq ($(ARCH),arm64)
-	BASEIMAGE?=arm64v8/debian:jessie
-else ifeq ($(ARCH),ppc64le)
-	BASEIMAGE?=ppc64le/debian:jessie
-else ifeq ($(ARCH),s390x)
-	BASEIMAGE?=s390x/debian:jessie
-else
-$(error Unsupported platform to compile for)
-endif
+BASEIMAGE?=gcr.io/google-containers/debian-base-$(ARCH):0.2
 
 GO_BUILD       = env GOOS=$(PLATFORM) GOARCH=$(ARCH) go build -i $(GOFLAGS) \
                    -ldflags "-X $(SC_PKG)/pkg.VERSION=$(VERSION) $(BUILD_LDFLAGS)"

